CONTRIBUTORS.md

```diff
@@ -0,0 +1,18 @@
+This plugin is part of the OpenTimelineIO project, whose contributors are
+listed at this URL:
+https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/CONTRIBUTORS.md
+
+This is a list of people who have contributed code to the
+otio-aaf-adapter project, sorted alphabetically by first name.
+
+If you know of anyone missing from this list, please contact us:
+https://lists.aswf.io/g/otio-discussion
+
+* Eric Reinecke ([reinecke](https://github.com/reinecke))
+* Josh Burnell ([JoshBurnell](https://github.com/JoshBurnell))
+* Joshua Minor ([jminor](https://github.com/jminor))
+* Julian Yu-Chung Chen ([jchen9](https://github.com/jchen9))
+* Mark Reid ([markreidvfx](https://github.com/markreidvfx))
+* Stefan Schulze ([stefanschulze](https://github.com/stefanschulze))
+* Stephan Steinbach ([ssteinbach](https://github.com/ssteinbach))
+* Tim Lehr ([timlehr](https://github.com/timlehr))
```

Comment:
Could you add these people also?
* Shahbaz Khan ([shahbazk8194](https://github.com/shahbazk8194))
* Freeson Wang ([freesonluxo](https://github.com/freesonluxo))
* Andrew Moore ([andrewmoore-nz](https://github.com/andrewmoore-nz))

Reply:
will do!